sujin-park/sorting/HIndex.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * 프로그래머스 - H-Index
+ * @param {number[]} citations 
+ * @return {number}
+ */
+function solution(citations) {
+    let result = 0;
+
+    citations.sort((a, b) => b - a);
+    citations.forEach((citation) => {
+        const count = citations.filter((item) => item >= citation).length;
```

Comment:
```suggestion
    citations.forEach((citation, index) => {
        const count = index + 1; 
```
이미 정렬은 했으니 갯수는 탐색없이 구할 수 있을 것 같아요! 수진님의 풀이에서 filter만 빠진게 다른 간결했던 풀이들이네요! 덕분에  이해됐습니다. 감사합니다.

Reply:
오 그렇네요 대박 감사합니다ㅎㅎㅎㅎ풀이 하나 더 추가했어요! 머지하겠습니당!